web_external/views/body/SubmissionView.js

```diff
@@ -1,23 +1,42 @@
-/**
-* View for an individual submission.
-*/
-covalic.views.SubmissionView = covalic.View.extend({
+import _ from 'underscore';
+import { getCurrentUser } from 'girder/auth';
+import { AccessType, DATE_SECOND } from 'girder/constants';
```

Comment:
`DATE_SECOND` is in `girder/misc`. Is there any way to detect errors like this other than at runtime?